plugins/wacom-usb/fu-wac-device.c

```diff
@@ -808,7 +809,18 @@ fu_wac_device_add_modules(FuWacDevice *self, GError **error)
 		st_item = fu_struct_module_item_parse(buf->data, buf->len, offset, error);
 		if (st_item == NULL)
 			return FALSE;
+
 		ver = fu_struct_module_item_get_version(st_item);
+		ver2 = fu_struct_module_item_get_version2(st_item);
+
+		/* When ver2 is available and not 0,          */
+		/* it is appended to ver in order             */
+		/* to make it BCD 32bits, other wise it stays */
+		/* BCD 16bits.                                */
```

Comment:
I think this would be more usual style to have:

		/*
		 * When ver2 is available and not 0, it is appended to ver in order to make it BCD
		 * 32bits, otherwise it stays BCD 16bits.
		 */